tools/Status/Status.py

```diff
@@ -308,25 +279,87 @@ def status_command(show_paths, show_unidentified, **kwargs):
                     table = rich.table.Table(*columns, box=None)
         if not show_paths:
-            console.print(table)
+            yield table
 
     # Output jobs that couldn't be parsed
     unidentified_jobs = job_data[job_data["ExecutableName"].isnull()]
     if len(unidentified_jobs) > 0 and show_unidentified:
-        console.print("")
-        console.rule("[bold]Unidentified Jobs", align="left")
+        yield ""
+        yield rich.rule.Rule("[bold]Unidentified Jobs", align="left")
         table = rich.table.Table(*standard_columns, box=None)
         for i, row in unidentified_jobs.iterrows():
             row_formatted = [
                 _format(field, row[field]) for field in standard_fields
             ]
             table.add_row(*row_formatted)
-        console.print(table)
+        yield table
+
+
+@click.command()
+@click.option("-u",
+              "--uid",
+              "--user",
+              "user",
+              show_default="you",
+              help=("User name or user ID. "
+                    "See documentation for 'sacct -u' for details."))
+@click.option("-a",
+              "--allusers",
+              is_flag=True,
+              help=("Show jobs for all users. "
+                    "See documentation for 'sacct -a' for details."))
+@click.option("-p",
+              "--show-paths",
+              is_flag=True,
+              help=("Show job working directory and input file paths."))
+@click.option("-U",
+              "--show-unidentified",
+              is_flag=True,
+              help=("Also show jobs that were not identified "
+                    "as SpECTRE executables."))
+@click.option("-s",
+              "--state",
+              help=("Show only jobs with this state, "
+                    "e.g. running (r) or completed (cd). "
+                    "See documentation for 'sacct -s' for details."))
+@click.option("-S",
+              "--starttime",
+              show_default="start of today",
+              help=("Show jobs eligible after this time, e.g. 'now-2days'. "
+                    "See documentation for 'sacct -S' for details."))
+@click.option("-w",
+              "--watch",
+              type=float,
+              default=None,
+              help=("On a new screen, refresh jobs every 'watch' number "
+                    "of seconds. Exit out with Ctl+C."))
+def status_command(show_paths, show_unidentified, **kwargs):
+    """Gives an overview of simulations running on this machine."""
+
+    # Start printing things
+    console = rich.console.Console()
+
+    refresh_rate = kwargs["watch"]
+
+    if not refresh_rate:
+        console.print(render_screen(show_paths, show_unidentified, **kwargs))
+        return
+
+    try:
+        with rich.live.Live(render_screen(show_paths, show_unidentified,
+                                          **kwargs),
+                            console=console,
+                            screen=True) as live:
```

Comment:
Add `auto_refresh=False` here and add `refresh=True` in the `update` call below. Otherwise you get weird interaction between the default refresh rate (4fps) and the one you specified manually.